tests/class/hash.c

```diff
@@ -329,34 +330,55 @@ void free_tree(node_t **tree)
     *tree = NULL;
 }
 
-static  void init_keys(uint64_t *keys, int nbkeys)
+static  void init_keys(uint64_t *keys, int nbkeys, int64_t seed, int structured_keys)
 {
-    node_t *tree = NULL;
     struct timeval start, end, delta;
     int print_end = 0;
     gettimeofday(&start, NULL);
     srand(start.tv_sec * 1000000 + start.tv_usec);
```

Comment:
I think this srand call is not needed anymore.

Reply:
Fixed